sdk/tools/spec2def/spec2def.c

```diff
@@ -1296,6 +1425,13 @@ int main(int argc, char *argv[])
     /* Zero terminate the source */
     pszSource[nFileSize] = '\0';
 
+    pexports = ParseFile(pszSource, file, &cExports);
+    if (pexports == NULL)
```

Comment:
Your new version only returns NULL in case of allocation failure. You'd want to check `cExports == 0` as well (and in that case free the buffer)

Reply:
We can't bail on cExports = 0, because we actually have empty spec files, but still need the generated files.